salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -141,7 +141,8 @@
                 var filter = ' p em strong strike u sub sup; a[!href](salsah-link)';
```

Comment:
The configuration of CKeditor, the filter and the standard mapping have to conform with one another.